signal2html/addressbook.py

```diff
@@ -217,11 +218,11 @@ def _load_recipients(self):
 
 
 class AddressbookV2(Addressbook):
-    def get_recipient_by_address(self, address: str) -> Recipient:
+    def get_recipient_by_address(self, address: Union[int, str]) -> Recipient:
```

Comment:
In which case were we 'improperly' passing an int?

Reply:
Yeah, I'll have to sort that out, this was admittedly a bit of a lazy fix. I've created a new issue (#40) to track this. Thanks for the check! :+1: